presto-native-execution/presto_cpp/main/tests/AnnouncerTest.cpp

```diff
@@ -12,12 +12,34 @@
  * limitations under the License.
  */
 #include "presto_cpp/main/Announcer.h"
+#include <boost/algorithm/string.hpp>
+#include <boost/filesystem.hpp>
 #include <gtest/gtest.h>
 #include "presto_cpp/main/tests/HttpServerWrapper.h"
 
+namespace fs = boost::filesystem;
 using namespace facebook::presto;
 
 namespace {
+
+std::string getCertsPath(const std::string& fileName) {
```

Comment:
Not sure if we could move this under tests/HttpTestUtil.h to share. Consider to refactor this in a followup. Thanks!